includes/class-wcs-importer.php

```diff
@@ -343,6 +351,8 @@ public static function import_subscription( $data ) {
 
 				if ( $set_manual ) {
 					$result['warning'][] = esc_html__( 'No payment method was given in CSV and so the subscription has been set to manual renewal.', 'wcs-import-export' );
```

Comment:
should this error message be not shown if the requires manual renewal flag is true as its not really a warning but a valid choice on the part of the user. or maybe another slightly different message when but both are true saying that manual renewal was selected